deployment/ansible/roles/cac-tripplanner.otp-data/defaults/main.yml

```diff
@@ -4,7 +4,5 @@
 #otp_data_dir: /var/otp
 otp_osm_source: https://s3.amazonaws.com/metro-extracts.mapzen.com/philadelphia_pennsylvania.osm.pbf
 otp_osm_filename: /var/otp/philadelphia.osm.pbf
-otp_gtfs_sources:
-  - septa_rail
-  - septa_bus
-  - patco
+otp_repo: "https://github.com/azavea/OpenTripPlanner.git"
```

Comment:
I was thinking we might want to create a CAC-specific branch on our fork for modifications, but as that hasn't proven necessary yet, I'll set it back to use the upstream branch/tag.